src/types.test.ts

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
@yusukebe in this file there are lots of unused variables and incorrect types, I wasn't sure if there is a specific reason for all of this so I just added this eslint-disable comment

If there isn't a specific reason for that let me know and I can do some cleanup here